inc/widget.php

```diff
@@ -93,7 +95,13 @@ function render_dashboard_widget() {
 
 			$body .= '<li class="entry">';
 			$body .= "<h3>$title</h3>";
-			$body .= $description;
+			if (DashboardChangelog\should_translate()) {
```

Comment:
nit: spacing, e.g.:

```suggestion
			if ( DashboardChangelog\should_translate() ) {
```

🔢

(I use the 🔢 emoji to indicate that there are other instances of the same issue in a file, in this case not putting a space inside parentheses.)

Reply:
Would you mind adding a linter ruleset to the repo?

Reply:
Yeah, I was actually thinking about that as I was writing the comment. I will get on that. 😁

Reply:
I can't run phpcs, it throws an error:
```console
PHP Fatal error:  Uncaught TypeError: vsprintf(): Argument #2 ($values) must be of type array, string given
```

Is it working on your side ?

Reply:
It was for me. Maybe a PHP version thing. I am running PHP8 locally.

Reply:
Nevermind, something was wrong in my setup, all is fine now